### ISSUE TITLE
Simplify the elements formatting

### ISSUE THREAD

**Issue by robUx4** [ietf-wg-cellar/matroska-specification#393]
In the EBML spec we don't use double quotes for all elements. For example
```
   EBMLMaxIDLength Element

   name: "EBMLMaxIDLength"

   path: "\EBML\EBMLMaxIDLength"

   id: "0x42F2"

   minOccurs: "1"

   maxOccurs: "1"

   range: "4"

   default: "4"

   type: "uinteger"
```

is in EBML
```
   EBMLMaxIDLength Element

   name: EBMLMaxIDLength

   path: "\EBML\EBMLMaxIDLength"

   id 0x42F2

   minOccurs: 1

   maxOccurs: 1

   range: >=4

   default: 4

   type: Unsigned Integer
```

**Comment by robUx4** [comment]
Fixed by #396

**Comment by robUx4** [comment]
#396 is merged so we can fix this issue
